exporter/sentryexporter/sentry_exporter.go

```diff
@@ -106,6 +109,8 @@ func (s *SentryExporter) pushTraceData(_ context.Context, td pdata.Traces) error
 
 	transactions := generateTransactions(transactionMap, orphanSpans)
 
+	transactions = append(transactions, exceptionEvents...)
```

Comment:
Well this is no longer only transactions. Can we rename to events? We’ll have to rename the transport method name was well, `CaptureEvent`?

Reply:
events sounds good, as far as the transport function is concerned, how about `SendEvents`?

Reply:
Yeah `SendEvents` works for me